src/services/groups.spec.ts

```diff
@@ -41,12 +41,16 @@ const wordlist: string[] = [
 describe('service: groups', () => {
   let dbPool: PostgresJsDatabase<typeof db>;
   let dbConnection: postgres.Sql<NonNullable<unknown>>;
+  let group: db.Group[];
 
   beforeAll(async () => {
     const initDb = createDbPool(DB_CONNECTION_URL, { max: 1 });
     await runMigrations(DB_CONNECTION_URL);
     dbPool = initDb.dbPool;
     dbConnection = initDb.connection;
+    const { groups } = await seed(dbPool);
+    // Insert registration fields for the user
```

Comment:
```suggestion
```

Reply:
I don't understand the comment

Reply:
ahh good catch! Was because I copied the seed and group code from another test